versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/aws.md

```diff
@@ -37,7 +37,7 @@ The AWS module just creates an EC2 KeyPair, an EC2 SecurityGroup and an EC2 inst
 
 1. Clone [Rancher Quickstart](https://github.com/rancher/quickstart) to a folder using `git clone https://github.com/rancher/quickstart`.
 
-2. Go into the AWS folder containing the terraform files by executing `cd quickstart/aws`.
+2. Go into the AWS folder containing the terraform files by executing `cd quickstart/rancher/aws`.
```

Comment:
Suggest "Terraform" (capitalized) over "terraform"

Reply:
Updated and applied to other versions of similar pages as well.